TombEngine/Game/camera.cpp

```diff
@@ -323,6 +332,131 @@ void MoveCamera(GameVector* ideal, int speed)
 	}
 }
 
+
+void ObjCamera(ItemInfo* CamSlotId, int CamMeshID, ItemInfo* TargetItem, int TargetMeshID, bool cond)
+{
+	GameVector from, to;
```

Comment:
No need to declare local variables at the start of the function. I believe that's a practice from the 80s when compilers needed it, but these days it adds a lot of clutter. Please initialise local variables on the first instance of use, like this:
`auto from = GameVector(dest, CamSlotId->RoomNumber);`